### ISSUE TITLE
CS3: Voice definitions and voice timing definitions should match used voice language, rather than used text language.

### ISSUE THREAD

**Issue by shinkiseki** [AdmiralCurtiss/SenPatcher#91]
Similar to #23. Currently playing through CS3 with EN text and JP voice, but the lip animations look like they better match the EN text than the JP voice. I can also check the other permutations of text and voice, but it's noticeable enough that I suspect that the wrong voice timing is the issue.

As I haven't gotten to CS4, I can't confirm whether this is also an issue there.

**Comment by AdmiralCurtiss** [comment]
Entirely plausible, yes.

**Comment by AdmiralCurtiss** [comment]
Hm, the presence of a `t_vctiming_jp.tbl` in `data/text/dat_en` suggests that they did at least consider this at some point, maybe the lipflap data is just bad. You can try coping that over `t_vctiming_us.tbl` and see if that improves the situation?

**Comment by shinkiseki** [comment]
Yes, replacing `t_vctiming_jp.tbl` with `t_vctiming_us.tbl` does the trick! The lip animations now match the JP voice properly.

I'm guessing this means that the game doesn't properly use the JP tbl file.

**Comment by AdmiralCurtiss** [comment]
Hmm, are you by chance using the 'continue from latest save' feature? What I'm seeing here suggests that a normal boot properly loads the correct voice timings, as does switching them in the Config menu, but if you use the 'quick-boot' from the latest save it defaults to the English ones and doesn't realize it needs to switch to the JP ones.

**Comment by shinkiseki** [comment]
Yes, I have been using 'continue from latest save', as a matter of fact.

**Comment by AdmiralCurtiss** [comment]
Apparently this got officially fixed in CS4 today according to the patch notes. Maybe CS3 too but there's no note about it?

**Comment by AdmiralCurtiss** [comment]
Confirmed fixed in CS4 and confirmed not fixed in CS3.
